presto-docs/src/main/sphinx/release/release-0.146.rst

```diff
@@ -13,6 +13,8 @@ Hive Changes
 * Fix incorrect skipping of data in Parquet during predicate push-down.
 * Fix reading of Parquet maps and lists containing nulls.
 * Fix reading empty ORC file with ``hive.orc.use-column-names`` enabled.
+* Fix unexpected exception from output stage JSON truncation in completion event
```

Comment:
This seems to be missing something here: "from output stage JSON trunction"

Or try to rephrase this. Also, end release notes in a period.